test/index.spec.ts

```diff
@@ -723,6 +842,116 @@ describe('[ TOTP ]', () => {
       expect(session.data.user.name).toBe('John Doe')
     })
   })
+
+  describe('End to End', () => {
```

Comment:
Love this! Also required one that could help other developers understand the flow of the strategy just by looking at the test!